Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.UnitTests/SnapshotHelper/ApiGatewayTestSnapshot.cs

```diff
@@ -0,0 +1,70 @@
+using System.Net;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Amazon.Lambda.TestTool.UnitTests.SnapshotHelper;
+
+public class HttpResponseMessageConverter : JsonConverter<HttpResponseMessage>
```

Comment:
have to use a cusotm json serializer here instead of default jsonconverter because HttpResponseMessage is cyclic in nature with some of its attributes so json cannot be easily created